mne/viz/_brain/tests/test_brain.py

```diff
@@ -312,6 +312,9 @@ def __init__(self):
     brain.add_foci([0], coords_as_verts=True,
                    hemi=hemi, color='blue')
 
+    # add aseg
+    brain.add_aseg(aseg='aseg')
```

Comment:
to speed this up can you set `rois` to some small list of one or two? This will also have a nice side-effect of hitting the `legend` lines